.vsts-ci.yml

```diff
@@ -82,8 +82,8 @@ stages:
     - template: eng/build.yml
       parameters:
         agentOs: Linux
-        jobName: Build_Fedora_39_Debug_x64
```

Comment:
Was the use to 39 intentional here?  Why the downgrade?

Reply:
The initial change was 36 -> 39. I intended to move to the oldest supported Fedora version, which is 38. 39 was a typo.